cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -156,6 +155,19 @@ void print_cumulative_row_info(host_span<cumulative_page_info const> sizes,
 }
 #endif  // CHUNKING_DEBUG
 
+compression_type from_parquet_compression(Compression compression)
+{
+  switch (compression) {
+    case Compression::BROTLI: return compression_type::BROTLI;
+    case Compression::GZIP: return compression_type::GZIP;
+    case Compression::LZ4_RAW: return compression_type::LZ4;
+    case Compression::SNAPPY: return compression_type::SNAPPY;
+    case Compression::ZSTD: return compression_type::ZSTD;
+    case Compression::UNCOMPRESSED: return compression_type::NONE;
+    default: CUDF_FAIL("Unsupported compression type");
+  }
+}
```

Comment:
nit: can this be turned into a host device function for reuse in `get_decomp_info` below

Reply:
Added a helper function to de-duplicate code. Did make the code a bit more convoluted.